lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -292,6 +292,18 @@ <h4 class="message-title">
 
 
   </section>
+  % if course_requirements:
+  <% prc_target = reverse('about_course', args=[course_requirements['prc_id'].to_deprecated_string()]) %>
```

Comment:
Don't use to_deprecated_string. Instead unicode(course_requirements['prc_id']). Make this change throughout.